### DISCUSSION TITLE
Rename "Disable Folding" and "Auto Unfold Foreign Scenes" to better match their true functionality

### DISCUSSION THREAD

**Discussion by samuelfine** [godotengine/godot-proposals#9268] [Editor]
The "Disable Folding" setting is named awkwardly compared to other Editor Settings. Specifically, it expects the user to perform an "affirmative" action (setting a checkbox to a checked / "yes" state) to apply a "negative" action (disabling folding behavior). For example, the option directly below (in Godot 4.3.dev4) is "Auto Unfold Foreign Scenes", which the user activates by checking a checkbox. If we applied similar grammar to this, the name would be something like "Don't Fold Foreign Scenes".

Furthermore, all other instances of "folding" in the Editor are related to code folding, _except_ for these two Settings. The tooltips for both these Settings also use the terms "expand" and "collapse" to describe the behavior, not "unfold" and "fold". Changing these names to match those descriptions should reduce user confusion by matching the names to the behavior, and reducing overlap between those and unrelated code editor functionality.

**Before:**
Disable Folding
Auto Unfold Foreign Scenes

**After: (recommendation)**
Expand All Property Groups
Auto Expand Foreign Scene Groups

**Comment by Calinou** [comment]
I agree with using "expand" terminology instead of "fold" here to avoid the confusion with code folding.

However, if we rename the setting to **Expand All Property Groups**, it becomes less obvious that you won't be able to fold sections in the inspector anymore when this setting is enabled. It's not just about expanding things by default; instead, things are *always* expanded with no way to collapse them.
